ui/app/styles/components/popup-menu.scss

```diff
@@ -48,6 +48,18 @@
     width: 100%;
   }
 
+  // TODO HDS polish - temp styling fix for ConfirmAction dropdown buttons
```

Comment:
How the popup menu styling looks - left is new component with this class, right is old component
<img width="1248" alt="Screenshot 2023-11-15 at 7 12 32 PM" src="https://github.com/hashicorp/vault/assets/68122737/ce4a4c32-1fca-4bd8-a4e6-c8da1cb5eef1">